modules/core/src/main/java/org/apache/ignite/internal/management/baseline/BaselineTaskResult.java

```diff
@@ -252,6 +266,7 @@ public boolean isBaselineAdjustInProgress() {
     /** {@inheritDoc} */
     @Override protected void writeExternalData(ObjectOutput out) throws IOException {
         out.writeBoolean(active);
+        out.writeObject(clusterState);
```

Comment:
Please use `U.writeEnum()` and `U.readEnum()` for ClusterState.

Reply:
Methods U.writeEnum() and U.readEnum() are now used for ClusterState